### DISCUSSION TITLE
Override duration to apply like normal limits instead of counting down automatically

### DISCUSSION THREAD

**Discussion by KL13NT** [proginosko/LeechBlockNG#466] [Ideas]
I want to use the override feature to give myself more time temporarily on a certain block. What I expect is:

- Override is set to 5 minutes
- I visit the website for 1 minute
- Close the tab
- Come back 5 hours later and find 4 minutes remaining in my override

Would love to see this. This discussion is based on https://github.com/proginosko/LeechBlockNG-chrome/issues/74

**Comment by proginosko** [comment]
I think you could get the same effect (or something close) by using two block sets. The first would block the sites 24/7 but allow overrides. The second (with the same sites) would allow some time limit (say 5 minutes) within a 24-hour period (but no overrides). The first block set would trump the second (i.e., sites completely blocked) unless an override were activated, in which case the complete block would be removed and the time limit would kick in.

Disclaimer: I haven't tested this!

**Reply by KL13NT** [reply]
That should work but that isn't what I'm asking here, sadly. What I'm looking for is essentially _adding_ time to today's limits (preferably a specific block set). This would just apply normal time limit when override is on.

**Reply by proginosko** [reply]
Sure, I understand you're requesting something different. I was just suggesting it as a workaround for now. It would take a bit of work to implement what you have in mind. It would have to be a feature separate from the override option, since it would work quite differently.

**Reply by KL13NT** [reply]
Do you imagine how it could be implemented then? I'm willing to PR it myself :)
